nautiluszim/scraper.py

```diff
@@ -19,7 +19,12 @@
 from zimscraperlib.fix_ogvjs_dist import fix_source_dir
 from zimscraperlib.inputs import handle_user_provided_file
 from zimscraperlib.i18n import setlocale, get_language_details, _
-from zimscraperlib.imaging import resize_image, get_colors, is_hex_color, create_favicon
+from zimscraperlib.image.probing import get_colors,is_hex_color
+from zimscraperlib.image.convertion import create_favicon
+from zimscraperlib.image.transformation import resize_image
+from zimscraperlib.constants import MAXIMUM_DESCRIPTION_METADATA_LENGTH,MAXIMUM_LONG_DESCRIPTION_METADATA_LENGTH
+
+
```

Comment:
Updated the import statements to fetch required methods from scraperlib version 3.0.0.
Imported the maximum length from constants.